### PR TITLE
Signs minor changes

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

File: src/signs/ui/search/SignsSearchForm.tsx [hunk 1]
```diff
@@ -33,11 +33,13 @@ function SignsSearchForm({ sign, signQuery, history }: Props): JSX.Element {
     if (unnormalizedSignQuery) {
       history.push(
         `?${stringify({
+          isComposite: false,
+          isIncludeHomophones: false,
           ...signQueryState,
```
Comment (yCobanoglu) [inline src/signs/ui/search/SignsSearchForm.tsx:38]: is more consistent default value of checkboxes is false.

File: src/signs/ui/search/MesZLContent.tsx [hunk 2]
```diff
@@ -40,51 +46,57 @@ export default function MesZlContent({
...
 
-  if (mesZlHead && mesZlBody) {
+  if (ready) {
     return (
       <>
         <div>
           <pre>
             <div
+              style={{ fontFamily: 'Assurbanipal' }}
               className="text-center"
```
Comment (yCobanoglu) [inline src/signs/ui/search/MesZLContent.tsx:69]: @jlaasonen  i will change it to a css class but i don't know why it isn't inheriting it from the body currently.

Comment (jlaasonen) [reply src/signs/ui/search/MesZLContent.tsx:72]: Maybe `pre` has overriding styles? You should be able to see where the final styles come from with the browser's dev tools.

File: .dockerignore [hunk 3]
```diff
@@ -1,3 +1,3 @@
 src/**/*.test.ts
-src/**/*.test.tsx
+src/**/*.test.ts
```
Comment (jlaasonen) [inline .dockerignore:2]: This looks erroneous. Tsx tests should be ignored and ts tests are already ignored above.

File: src/signs/ui/search/MesZLContent.tsx [hunk 4]
```diff
@@ -7,63 +7,68 @@ import remark2rehype from 'remark-rehype'
...
+import { Link } from 'react-router-dom'
 
-function splitMesZl(mesZl: string): { mesZlHead: string; mesZlBody: string } {
+function splitMesZl(
+  mesZl: string
+): { mesZlHeadMarkdown: string; mesZlBodyMarkdown: string } {
   const mesZlLines = mesZl.split('\n')
   const mesZlBody = mesZlLines
-    .slice(1, 6)
+    .slice(1, 7)
```
Comment (jlaasonen) [inline src/signs/ui/search/MesZLContent.tsx:17]: Extract constant for the magic number.

File: src/signs/ui/search/Signs.css [hunk 5]
```diff
@@ -19,7 +19,10 @@
   border-style: solid;
   border-width: 0 0 0.5px 0;
 }
-
+.HangingIndent {
+  padding-left: 44px;
+  text-indent: -44px;
+}
```
Comment (jlaasonen) [inline src/signs/ui/search/Signs.css:25]: The coding conventions tell to use BEM naming. But they have been violated many times. Do you think we should drop the convention(s)?

File: src/signs/ui/search/MesZLContent.tsx [hunk 6]
```diff
@@ -8,28 +8,34 @@ import raw from 'rehype-raw'
...
+
+async function convertMarkdownToHtml(markdown: any): Promise<string> {
   const subSup = (mesZL: string): string =>
     mesZL
       .replace(/\^([^\^]*)\^/g, '<sup>$1</sup>')
       .replace(/~([^~]*)~/g, '<sub>$1</sub>')
 
+  const italic = (mesZL: string): string =>
+    mesZL.replace(/\*([^*]*)\*/g, '<em>$1</em>')
+
```
[the remark this answers is not in the document]
Comment (jlaasonen) [reply src/signs/ui/search/MesZLContent.tsx:32]: Please add a comment.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: src/signs/ui/search/MesZLContent.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (yCobanoglu) [reply src/signs/ui/search/MesZLContent.tsx:32]: our markdown -> html transformer library uses commonMarkdown and there are occurences of which are not valid common Markdown which are parsed manually.

File: src/signs/ui/search/Signs.css (hunk 5 above)
Comment (yCobanoglu) [reply src/signs/ui/search/Signs.css:25]: no i am going to read up again on them.

File: src/signs/ui/search/Signs.css [hunk 7]
```diff
@@ -1,44 +1,47 @@
...
   line-height: 180%;
 }
 
-.MesZL--popover {
+.mesZL__popover {
   min-width: 40%;
 }
-.MesZL--popover p {
+.mesZL__popover pre {
+  font-family: Assurbanipal, Junicode, serif !important;
```
[the remark this answers is not in the document]
Comment (yCobanoglu) [reply src/signs/ui/search/Signs.css:14]: removed it

**Review by jlaasonen — APPROVED** [review]

**Pull request merged**